.github/pull_request_template.md

```diff
@@ -0,0 +1,3 @@
+:stop_sign: Don't merge anything into `master`, please use `dev`.
```

Comment:
Remove this commit, no relation to this branch. Do this in another pull request.